siac/rentercmd.go

```diff
@@ -353,9 +398,15 @@ func renterfilesdownloadcmd(path, destination string) {
 // SiaPath field.
 type bySiaPath []modules.FileInfo
 
-func (s bySiaPath) Len() int           { return len(s) }
-func (s bySiaPath) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
-func (s bySiaPath) Less(i, j int) bool { return s[i].SiaPath < s[j].SiaPath }
+func (s bySiaPath) Len() int {
+	return len(s)
+}
+func (s bySiaPath) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+func (s bySiaPath) Less(i, j int) bool {
+	return s[i].SiaPath < s[j].SiaPath
+}
```

Comment:
it's convention to keep these methods all on one line because they are concise and tend to be (nearly) the same for all `sort.Sort` implementers.
If you are interested, now that we have upgraded to Go 1.8, we can replace a fair amount of our sorting types with [`sort.Slice`](https://golang.org/pkg/sort/#Slice).